system/lib/compiler-rt/lib/asan/asan_flags.cpp

```diff
@@ -129,7 +129,7 @@ void InitializeFlags() {
   char *options;
   // Override from Emscripten Module.
 #define MAKE_OPTION_LOAD(parser, name) \
-    options = (char*) EM_ASM_INT({ \
+    options = (char*)(long)EM_ASM_DOUBLE({ \
```

Comment:
Is this assuming that the pointer fits in 53 bits?

This seems worth at least a scary comment,
```cpp
// TODO: add EM_ASM_I64 and avoid using a double for a 64-bit pointer.
```

Reply:
We use pointers inside Numbers in a LOT of places, not just here. But sure.